.circleci/config.yml

```diff
@@ -260,9 +260,9 @@ workflows:
             - 'build-mac':
                   requires:
                       - unit-test
-            #            - 'e2e-mac':
-            #                  requires:
-            #                      - build-mac
+            - 'e2e-mac':
+                  requires:
+                      - build-mac
             - 'stage-mac':
                   requires:
                       - build-mac
```

Comment:
@eduardosmoura can you change to only stage-mac if e2e-mac passes. You need just to change the `build-mac` to `e2e-mac`

Also can you add a job to run e2e on linux and also require it to stage-linux?

Cheers

Reply:
Will do @rodrigopavezi